plutus-tx/test/Plugin/Functions/Spec.hs

```diff
@@ -117,3 +118,9 @@ mapDirect f l = case l of
 
 polyMap :: CompiledCode ([Integer])
 polyMap = plc @"polyMap" (mapDirect (Builtins.addInteger 1) [0, 1])
+
+myDollar :: (a -> b) -> a -> b
```

Comment:
`myDollar` doesn't seem to be used anywhere (and I would have expected the examples to use `$` as well)

Reply:
:sweat_smile: :sweat_smile: :sweat_smile: 

Forgot to actually call the function :grimacing:

Reply:
I'm not importing the Plutus Tx prelude here to avoid making things confusing, so here I'm just defining the same function locally with a non-clashing name for the purposes of testing.

Test fixed!